### DISCUSSION TITLE
The plans for DAOstar (season 2)

### DISCUSSION THREAD

**Discussion by thelastjosh** [metagov/daostar#183] [Announcements]
These are the plans for DAOstar Season 2 (i.e. Q1 - Q2). They were based on discussions with the core staff, at community calls, as well as the mission & vision discussions for DAOstar at the roundtable. See [typeform report](https://metagov.typeform.com/report/TVnZMm0x/c1iGWMNWfovqQfTR).

# Main projects
1. **DAOstar Data Service**, led by Amandeep and/or a future technical lead, focused on improving the indexing and data quality of DAO data published through EIP-4824 and related standards. This will involve working closely with DAO frameworks, DAO aggregators, and DAOs. See #25 
1. **DAOstar Regulatory Interoperability** (a.k.a. Millennium Falcon), led by Joni Pirovich, focused on defining a regulatory API / interface so that we can get legal benefits, especially limited liability status. This will eventually transition into a working group on tax and accounting standards.
2. **DAOstar Grants Management**, led by Eugene Leventhal and Joshua Tan, working with Gitcoin, Optimism, and others to define a common grants system standard for DAOs, Web3 foundations, and non-blockchain entities (i.e. normal foundations). This is timed with the recent Gitcoin protocol launch and the [Grant Innovation Lab](https://grant.metagov.org) led by Eugene.
3. **DAOstar Attestations** (a.k.a. Voyager), led by Joshua Tan, focused on building an interoperable (and minimal!) attestations system for DAOs and DAO tooling, compatible with SBTs, DIDs, verifiable credentials, and other attestation tooling.
4. **DAOstar Membership Program**, led by Hazel Devjani and/or other community lead. This is an evolution of the DAOstar roundtable and should be set up and tested _before the April roundtable / potential bull run_. Instead of getting one US-focused roundtable, can we convert the roundtable into a regional roundtable with LATAM, Asia, China, Europe, US, etc.?
5. **DAOstar Asia Roundtable**, led by Hazel Devjani, to develop the DAO ecosystem in Asia and foster new standards and public goods (likely a marketing survey). This may include a new Asia-focused fellowship program in discussion with 4Seas.
6. **DAOstar Developer Community and Ecosystem**, led by Hazel Devjani and/or other community lead. DAOstar should be THE destination for DAO builders.
7. Deepening our relationships with key players in blockchain and DAO infrastructure, e.g. Optimism and Arbitrum, led by Amandeep.
9. [Pending funding and capacity] **DAOstar Policy Proposals**, to cover best-practices and other community publications not in scope for the DAOIP system.
10. [Pending funding and capacity] DAOstar Governor Ecosystem Standards with Tally, OpenZeppelin, ScopeLift, Nouns, Joke DAO.
11. [Pending funding and capacity] DAOstar Delegations, working with folks at Karma, Agora, Tally, Uniswap, and Optimism to define a delegate registry and standard schemas for delegation.

# Active grants
1. Optimism for DAO adoption on Optimism #99 
2. Optimism for attestations framework #100 
5. ENS 10k for ENS / DNS integration #181 
8. Arbitrum for Arbitrum adoption + regulatory interop standard #184 
10. (potentially) EU NGI Sargasso for regulatory interop standard #171 
4. Optimism academic round (this is less DAOstar and more Constitutions of Web3 + DAO NYC events)

# Technical milestones
1. More infrastructure development, including a new frontend, better indexing, multichain support (i.e. we already have Juno DAOs getting indexed), deeper indexing (so displaying not just the URIs but also retaining the membership rolls), and historical data.

## [Milestone 2: key infrastructure](https://github.com/metagov/daostar/milestone/7) (done by January)
- versioning for registration contracts
- deploy a new registration contract
- docs, articles, walkthroughs, and other content; docs for additional frameworks & reference implementations
- indexing (DAODAO, Aragon, KALI, etc.)

## [Milestone 2.1: Attestations](https://github.com/metagov/daostar/milestone/6) (done by January)
- attestations infrastructure
- EAS integrations

## [Milestone 2.2: website](https://github.com/metagov/daostar/milestone/3) (done by end of Q1)
- consolidated daostar.org improvements #102 , including manager flow for updating daoURI
- test, extend, and iterate the flow based on actual customer feedback + discovery

## [Milestone 3: data indexing, data quality](https://github.com/metagov/daostar/milestone/2)
- data quality and backend upgrades to support adoption by DeepDAO, Messari, Etherscan
- deep Indexing (e.g. crawling actual member data and providing historical access to large providers or requestors), need more upgrades to the data
- source additional requirements from DeepDAO, Messari, Etherscan

## Milestone 3.1+: other features
- Indexing for regulatory API (see #114 )
- Indexing for grants management, e.g. indexing all available grants compliant with the grants management standard
- Additional indexing and support for attestations, including issuerURI integration (e.g. bringing on Hats, incorporating more of this into the management / frontend interface)

For previous technical milestones, see #52.

**Comment by amanwithwings** [comment]
Nooooo my name isn't mentioned in any of the initiatives😭😭😭😭

**Reply by thelastjosh** [reply]
Added you :D

**Comment by amanwithwings** [comment]
Let's bring more end-user adoption in season 2! In this regard, the attestation standard in its present or future forms, the Grants Innovation Lab, and EIP-4824 [assuming we can get a decent number of large DAOs (say 50+) to adopt it] are quite promising.   

As the growth lead, my short-term targets are: 
- adapting DAOIP-3 to EAS and ensuring its adoption among issuers,
- getting ENS, OP Collective, and 3 large DAOs on Arbitrum to publish daoURIs,
- becoming an active delegate at OP Collective,
- completing the Arbitrum Foundation Grant (getting the 80 DAOs they suggested for the bonus), and
- working with Hazel to roll out the membership proposal and with Eugene to support GIL. 

I would also like to work on a piece of research and/or establish a new working group if there is enough interest within the ecosystem for us to pursue a new direction.

**Comment by thelastjosh** [comment]
In a call with @frm , developing an overview of the entire technical stack and issue progress at the moment. Thought I'd share this with the entire group so as to not re-duplicate effort.

Key people involved right now:
- On technical side, it's Rashmi, Kahn and Mike (and Connie) from LXDAO, a service DAO / dev shop based in China / Asia, and Josh. Aman helps out on a lot of the projects not with code but by clarifying and driving requirements.
- On the standards authoring side, it's Joni, Balazs, Eugene
- On the community side, it's Hazel, Hisashi, and Joseph
- On the website / frontend side, it's Jenny and Jack (who also developed the new metagov.org)

With respect to the outstanding issues:

Milestone 2:
- #186 honestly I have no idea where that is, though I think we've at least improved our documentation of where our factory contracts actually are (across all the chains we support)
- #58 we've been plugging away at this slowly; Rashmi recently shipped a first version to docs.daostar.org using docusaurus, Aman has been helping her with this
- #187 is on hold due to Aragon. Will 
- #208 Josh has no idea.

Milestone 2.1:
- main relevant active one is #205 . Note also relevant issue in Snapshot's repo https://github.com/snapshot-labs/snapshot-hub/issues/825
